### ISSUE TITLE
I see no object detected after ~10000 training steps

### ISSUE THREAD

**Issue by Quykiemsau** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#32]
I'm trying to detect teeth from x-ray images, but after training about 10k steps, the loss is still around 1 and doesn't seem to be decreasing. Sometimes, it jumped up to 100000 and decreased after that. And after 10000 steps I tried to run image detection on some photos but it could not detect anything. Is there something that I have to configure? Thank you very much.
